3rd-party/crsqlite-wasm/scripts/build-wasm.sh

```diff
@@ -9,12 +16,15 @@ mkdir -p dist
 export CRSQLITE_COMMIT_SHA="$(git rev-parse HEAD)"
 
 cd ../emsdk
-./emsdk install 3.1.45
+# We redirect stderr to stout (with `2>&1`) so that rush doesn't think there were any warnings:
+# in case it detects any it will fail the build
+./emsdk install 3.1.45 2>&1
 ./emsdk activate 3.1.45
-source ./emsdk_env.sh
+export PATH=$PWD/upstream/emscripten:$PATH
 
 cd ../wa-sqlite
-make
+# We redirect stderr to stout (with `2>&1`) so that rush doesn't think there were any warnings:
+# in case it detects any it will fail the build
+make 2>&1
```

Comment:
Just to clarify...this is for the CI build (not the local one), the local one succeeds for me by simply printing out 'succeeded with warnings'

Reply:
Both locally and in the CI `rush build` exits non 0 in case of something in stderr.

But locally you can just ignore it, the CI marks the step as failed and acts accordingly.